pkg/internal/checkup/checkup.go

```diff
@@ -786,6 +813,13 @@ func (c *Checkup) Config() config.Config {
 
 func (c *Checkup) checkVMIBoot(ctx context.Context, errStr *string) error {
 	log.Print("checkVMIBoot")
+
+	if c.defaultStorageClass == "" {
```

Comment:
here you dont need to also check: `&& c.checkupConfig.StorageClass == ""` ?

Reply:
Right!